inc/commonitilobject.class.php

```diff
@@ -8239,4 +8239,31 @@ public static function getOpenCriteria(): array {
          ]
       ];
    }
+
+   public function displayHiddenItemsIdInput(array $options): void {
+      foreach ($options['items_id'] ?? [] as $itemtype => $items) {
+         foreach ($items as $items_id) {
+            echo "<input type='hidden' name='items_id[$itemtype][$items_id]' value='$items_id'>";
+         }
+      }
+   }
+
+   public function handleItemsIdInput(): void {
+      if (!empty($this->input['items_id'])) {
+         $item_class = static::getItemClass();
+         $item_ticket = new $item_class();
```

Comment:
Maybe `$item_ticket` should be renamed to ... (no idea, but is not only applies to tickets).

Reply:
Done in c962154.